packages/client/hmi-client/src/components/dataset/tera-save-dataset-from-simulation.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+	<Button
+		:disabled="isSaveDisabled"
+		outlined
+		title="Saves the current version of the model as a new Terarium asset"
+		@click="showSaveInput = !showSaveInput"
+	>
+		<span class="pi pi-save p-button-icon p-button-icon-left"></span>
+		<span class="p-button-text">Save as new dataset</span>
+	</Button>
+	<span v-if="showSaveInput" style="padding-left: 1em; padding-right: 2em">
+		<InputText v-model="saveAsName" class="post-fix" placeholder="New dataset name" />
```

Comment:
Put styles in class for the span. Also this `post-fix` class isn't in the css, maybe you forgot to move it?

Reply:
Thanks